content/blog/2019-05-09-tradestatistics.md

```diff
@@ -0,0 +1,280 @@
+
+The web service is [nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) enhanced with a secured connection by using [Let's Encrypt](https://www.digitalocean.com/community/tutorials/how-to-use-certbot-standalone-mode-to-retrieve-let-s-encrypt-ssl-certificates-on-ubuntu-1804). The landing page is a modified [HTML5UP](https://html5up.net/) template, and [Atom](https://atom.io/) and [yui-compressor](https://yui.github.io/yuicompressor/) were the tools to personalize the CSS behind the landing, documentation and dashboard with [Fira Sans](https://www.fontsquirrel.com/fonts/fira-sans) as the typeface of choice.
+
+Even our email service is powered thanks to Open Source. We use [mail-in-a-box](https://mailinabox.email/) with some very simple tweaks such as email forwarding and integration with Thunderbird.
+
+### rOpenSci contributions
+
+Thanks to [Maelle](https://github.com/maelle), [Amanda](https://github.com/aedobbyn), [Jorge](https://github.com/cimentadaj), [Emily](https://github.com/emilyriederer), [Mark](https://github.com/mpadge) the overall result can be said to be top quality!
+
+After a long reviewing process (more than six month considering initial submission!), what started as an individual process mutated into something that I consider a collective result. Thanks to the amazing team behind rOpenSci, to their constructive feedback, exhaustive software reviewing and the confidence to propose ideas that that I had never gotten, what you have now is not just a solid R package.
+
+The hours spent as a part of the reviewing process translated into changes to the database and the API. According to the reviewers comments, there are limited opportunities to implement server-side changes and then updating the R code. With the inclusion of different API parameters that I initially didn't consider, the current API/package state provides an efficient solution way better than post-filtering. You'll always extract exactly the data you require and no more than that.
+
+One useful contributed idea was to create aliases for groups of countries. Both in the API and package, you can request an ISO code such as "usa" (United States) or an alias such as "c-am" (all countries in America) that returns condensed queries.
+
+The only aspect we didn't totally agree was the dependency-heavy "tidy" code. I was asked for a lot of changes to the initial package draft, and one of the very few points that remained less changed was to use tidyverse functions, such as those provided by rlang package, instead of dependency-free base R alternatives. However, it's a matter of personal taste.
```

Comment:
This feels like too many words that take attention away from the main post. 

When you review the post, consider what you want your key points to be.